Sources/Frontend/src/views/BoardPage.vue

```diff
@@ -0,0 +1,99 @@
+<template>
```

Comment:
Pls use template for any html elements that will be inserted, 
No `document.createElement('canvas');` use v-if statements if needed